src/main/ui/src/components/SearchByState/SearchByState.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import {useState} from 'react'
+
+export default function SearchByState() {
+    const [data, setData] = React.useState([]);
+    const [searchTerm, setSearchTerm] = useState('');
+    React.useEffect(() => {
+    let url = `https://developer.nps.gov/api/v1/parks?parkCode="all"&limit=469&api_key=JMZizGv6gAcjBzoD4TbqW9RQSe9K8fEt9Cdb2Zta`;
+    fetch(url)
+        .then((response) => response.json())
+        .then((json) => {
+            setData(json.data);
+        })
+    .catch((error) => console.log(error));
+}, [searchTerm]);
+
+    function handleSearch(e) {
+        e.preventDefault();
+        setSearchTerm(e.target.value);
+    }
+
+    if (data.length > 0) {
+        data.filter((i) => {
+            return i.states.match(data);
+        });
+    }
+
+    function filterParkStates(park) {
+        if (searchTerm === "") {
+            return park
+        } if (park.states.toLowerCase().includes(searchTerm.toLowerCase())) {
+            return park
+        }
+    }
+
+return (
+<div>
+    <input
```

Comment:
In the requirements in the story and in discussion we had talked about making this a dropdown. Why did we not do that?